config/clusters/2i2c/ucmerced-staging.values.yaml

```
@@ -0,0 +1,76 @@
+jupyterhub:
+  ingress:
+    hosts:
+      - staging.ucmerced.2i2c.cloud
+    tls:
+      - secretName: https-auto-tls
+        hosts:
+          - staging.ucmerced.2i2c.cloud
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "google"
+    homepage:
+      templateVars:
+        org:
+          name: University of California, Merced
+          url: http://www.ucmerced.edu/
+          logo_url: https://brand.ucmerced.edu/sites/brand.ucmerced.edu/files/images/ucm-logo-text.png
+        designed_by:
+          name: 2i2c
+          url: https://2i2c.org
+        operated_by:
+          name: 2i2c
+          url: https://2i2c.org
+        funded_by:
+          name: University of California, Merced
+          url: http://www.ucmerced.edu/
+  singleuser:
+    profileList:
+      - display_name: Python
+        description: Start a Python server with JupyterLab and scientific Python stack
+        default: true
+        kubespawner_override:
+          image: jupyter/scipy-notebook:2023-06-26
+          # Launch into JupyterLab after the user logs in
+          default_url: /lab
+      - display_name: R
+        description: Start a R server with RStudio and tidyverse & Geospatial tools
+        kubespawner_override:
+          image: rocker/binder:4.3
+          default_url: /lab
+          # Ensures container working dir is homedir
+          # https://github.com/2i2c-org/infrastructure/issues/2559
+          working_dir: /home/rstudio
+          # Because this is a list, it will override our default volume mounts
+          volume_mounts:
+            # Mount the user home directory
+            - name: home
+              mountPath: /home/rstudio
+              subPath: "{username}"
+            # Mount the shared readonly directory
+            - name: home
+              mountPath: /home/rstudio/shared
+              subPath: _shared
+              readOnly: true
+  hub:
+    config:
+      Authenticator:
+        admin_users:
+          - schadalapaka@ucmerced.edu
+      JupyterHub:
+        authenticator_class: cilogon
+      CILogonOAuthenticator:
+        oauth_callback_url: https://staging.ucmerced.2i2c.cloud/hub/oauth_callback
+        shown_idps:
+          - urn:mace:incommon:ucmerced.edu
+          - https://accounts.google.com/o/oauth2/auth
+        allowed_idps:
+          urn:mace:incommon:ucmerced.edu:
+            username_derivation:
+              username_claim: "eppn"
+          http://google.com/accounts/o8/id:
+            username_derivation:
+              username_claim: "email"
+            allowed_domains:
+              - "2i2c.org"
```

- relocated `Authenticator` config to the bottom to be consistent in how other config is ordered
- removed `shown_idps` for oauth16
- added `allow_all` for the ucmerced.edu idp for oauth16 equivalent config

```suggestion
      JupyterHub:
        authenticator_class: cilogon
      CILogonOAuthenticator:
        oauth_callback_url: https://staging.ucmerced.2i2c.cloud/hub/oauth_callback
        allowed_idps:
          urn:mace:incommon:ucmerced.edu:
            username_derivation:
              username_claim: "eppn"
          http://google.com/accounts/o8/id:
            username_derivation:
              username_claim: "email"
            allowed_domains:
              - "2i2c.org"
      Authenticator:
        admin_users:
          - schadalapaka@ucmerced.edu
```

Related to #3144 being merged or not, and I'm looking to merge it in 30 minutes or so after reviewing all config once more

---

Thanks @consideRatio! I'll wait for the merge (🚀 🚀 🚀 ) to rebase this and manually deploy to double check config.